src/app/(main)/StatusChip.tsx

```diff
@@ -0,0 +1,41 @@
+import { Chip } from '@mui/material';
+import { Circle } from '@mui/icons-material';
+import { ResponderStatus } from '@respond/types/activity';
+
+export const StatusChip = ({ status }: { status: ResponderStatus }) => {
+
+    switch (status) {
```

Comment:
Maybe a place for a couple lookup tables? Outside of the class:
```
const STATUS_COLORS = { ResponderStatus.SignedIn: 'success', ResponderStatus.SignedOut: 'error', ... };
const STATUS_TEXT = { ResponderStatus.SignedIn: 'Signed In', ... };
```
and then:
```
return (<Chip icon={<Circle color={STATUS_COLORS[status]} /> ... />);
```

Reply:
I had something similar originally, but TS was throwing an error. I couldn't find a resolution, so I just assigned them directly.

Any ideas on a better way to resolve this type mismatch?

![image](https://github.com/KingCountySAR/respond-next/assets/11284884/ec0d4e3d-eb6a-47fd-a869-65d5499f8716)
![image](https://github.com/KingCountySAR/respond-next/assets/11284884/066b6b1b-12c7-4b0f-8895-eec47c518952)

Reply:
To clarify -- even though this type checking error exists. It does not cause a runtime error.

Reply:
Ah, yes. We can explicitly define the types for these maps, instead of relying on TS inference. Pushed that as a commit.